src/tests/unittests/utils/storage/test_storage_memory.py

```diff
@@ -65,6 +65,14 @@ def test_list_subtags(self):
         results = self.storage.list_data_subtags(['1a', '2', '3'])
         self.assertEqual(results, [])
 
+    def test_list_subtags_limit(self):
+        self.storage.save_data('1', ['a', '1'])
+        self.storage.save_data('1', ['a', '2'])
+        tags = self.storage.list_data_subtags(['a'], limit = 0)
+        self.assertEqual(tags, ['1', '2'])
+        tags = self.storage.list_data_subtags(['a'], limit = 1)
```

Comment:
Same here, spaces around =
```suggestion
        tags = self.storage.list_data_subtags(['a'], limit=1)
```